Formula/unisonlang.rb

```diff
@@ -21,23 +21,27 @@ class Unisonlang < Formula
   depends_on "haskell-stack" => :build
   depends_on "node" => :build
 
-  # stack/ghc currently have a number of issues building for aarch64
-  # https://github.com/unisonweb/unison/issues/3136
-  depends_on arch: :x86_64
-
   uses_from_macos "xz" => :build
   uses_from_macos "zlib"
 
   resource "codebase-ui" do
-    url "https://github.com/unisonweb/codebase-ui/archive/refs/tags/release/M3.tar.gz"
-    sha256 "84be9135a821615f8fc73a0a894aa46a11c55393c43dc26e16a5ce75f8063012"
-    version "M3"
+    url "https://github.com/unisonweb/unison-local-ui/archive/4d196d5d775c36490527552b3d98c929fc178a37.tar.gz"
```

Comment:
I thought versions here were supposed to match?

Reply:
They hadn't tagged this repo yet, but they just tagged it this morning. Fixed now.